json2csv.js

```diff
@@ -0,0 +1,278 @@
+  var excerptRows = 7;
+  var input;
+  var url;
+  var lastSaved;
+
+  // function log(msg) {
+  //   return $(".console").removeClass("error").html(msg);
+  // }
+
+  // function error(msg) {
+  //   return log(msg).addClass("error");
+  // }
+
+  function doJSON() {
+    // just in case
+    $(".drop").hide();
+
+    // get input JSON, try to parse it
+    var newInput = $(".json textarea").val();
```

Comment:
this is where I need help pulling in $type.json files... (entries.json, as example for asingle)